tgbot/src/methods/chat/set_photo.rs

```diff
@@ -10,10 +13,9 @@ use serde::Serialize;
 ///
 /// Note: In regular groups (non-supergroups), this method will only work
 /// if the ‘All Members Are Admins’ setting is off in the target group
-#[derive(Clone, Debug, Serialize)]
+#[derive(Debug)]
```

Comment:
Return `Clone` derives for structures

Reply:
+1, might be useful, the rule of thumb is to add `Clone` to all public types

Reply:
You cant clone because of Box<Read + Send>

Reply:
^ retracted